notebook/Dockerfile

```diff
@@ -39,8 +39,9 @@ RUN mamba install --yes python=${PYTHON_VERSION} nomkl \
     ipywidgets \
     cachey \
     streamz \
-    dask-labextension>=5 \
-    && mamba clean -tipsy \
+    "dask-labextension>=5"
```

Comment:
I doubt this was causing us much errors in the past, but escaping this should give us the outputs of the `mamba` install and also ensure that `dask-labextension>=5` (before it was just piping the output of the command to a file named `=5`)

Reply:
> before it was just piping the output of the command to a file named =5

Whoops, nice catch @charlesbluca

Reply:
Good catch!